src/storage-client/src/source/persist_source.rs

```diff
@@ -295,3 +305,44 @@ impl PendingWork {
         true
     }
 }
+
+#[derive(Debug)]
+struct PersistSourceDataStatsImpl<'a> {
+    stats: &'a PartStats,
+}
+
+impl PersistSourceDataStats for PersistSourceDataStatsImpl<'_> {
+    fn len(&self) -> Option<usize> {
+        None
+    }
+
+    fn err_count(&self) -> Option<usize> {
```

Comment:
What's the plan for implementing this one? (And how do we expect folks to use a raw update count?)

Reply:
Mostly just trying to figure out what this implies for the stats collection! But I suppose if we're counting both overall updates and null values for each column, I can see how a count of the non-null values in the error column falls out of that pretty nicely...